ext/opentelemetry-ext-wsgi/tests/test_wsgi_middleware.py

```diff
@@ -108,7 +108,9 @@ def validate_response(self, response, error=None):
                 self.span_context_manager.__exit__.assert_not_called()
                 self.assertEqual(value, b"*")
             except StopIteration:
-                self.span_context_manager.__exit__.assert_called()
+                self.span_context_manager.__exit__.assert_called_with(
```

Comment:
Too bad we need this change. 🙁 Which Python version requires it?

Reply:
We need it for 3.5 and 3.4.
The [assert_called](https://docs.python.org/3.6/library/unittest.mock.html#unittest.mock.Mock.assert_called) method was introduced in 3.6.